client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -669,4 +684,15 @@ function isNotBlocked( plugin ) {
 	return PLUGIN_SLUGS_BLOCKLIST.indexOf( plugin.slug ) === -1;
 }
 
+/**
+ * Returns a boolean indicating if a plugin is already installed or not
+ *
+ * @param plugin plugin object to be tested
+ * @param installedPlugins list of installed plugins aggregated by plugin slug
+ * @returns Boolean weather a plugin is not installed on not
+ */
+function isNotInstalled( plugin, installedPlugins ) {
+	return ! installedPlugins[ plugin.slug ];
+}
```

Comment:
This doesn't work in my site. `plugin.slug  = woocommerce-subscriptions` but `installedPlugins` contains a `woocomerce-com-woocommerce-subscriptions` key.

We had tackled this problem again in the past in https://github.com/Automattic/wp-calypso/pull/60627

Reply:
I'm not able to reproduce it, the `Woocomerce Subscriptions` plugin is being correctly filtered out of my list. 

Your suggestion would replace the slug check by the `isEqualSlugOrId` method?

Reply:
Yes, pushed a [fc190a3](https://github.com/Automattic/wp-calypso/pull/63712/commits/fc190a3bf52fe2b3b059b564d3968da227041d97) that seems working for me. Woocommerce subscriptions and xero do not appear now

<img width="1124" alt="CleanShot 2022-05-18 at 10 46 26@2x" src="https://user-images.githubusercontent.com/12430020/168985818-a81b80a1-9d82-4484-acc7-f6d7df6686dd.png">

Reply:
Great, thanks for handling this.